lib/SILOptimizer/Utils/ValueLifetime.cpp

```diff
@@ -135,8 +135,15 @@ bool ValueLifetimeAnalysis::computeFrontier(Frontier &frontier, Mode mode,
         continue;
       }
       // In case the last user is a TermInst we add all successor blocks to the
-      // frontier (see below).
-      assert(deadInSucc && "The final using TermInst must have successors");
+      // frontier (see below). Except it is a function exiting TermInst, like
+      // 'return' or 'throw'.
```

Comment:
Can you expand the comment here why the term inst is an exception. I understand why, it should just be explicit in the code.